test/integration/backends/ibis/test_end_2_end.py

```diff
@@ -64,9 +65,18 @@ def make_ibis_db(db_conn, metadata_conn, tmp_dir, in_memory=False):
     )
 
 
+testdata = [(test_db), (ibis_pandas_db)]
+
+
 @pytest.mark.skipif(DO_SKIP, reason="skipping ibis tests if mongodb")
-def test_end_2_end(test_db, memory_table=False):
-    db = test_db
+def test_end_2_end(test_db, ibis_pandas_db):
+    # SQL - Non memory table (sqlite)
+    _end_2_end(test_db)
+    # SQL - memory table (Pandas)
+    _end_2_end(ibis_pandas_db, memory_table=True)
```

Comment:
Please follow the design of the other data-backend testing.

Reply:
mongodb and local are only tested with artefacts I guess in this test module